en/messages.json

```diff
@@ -65,6 +65,33 @@
     "popupSettingsTermsOfService": {
         "message": "Terms of Service"
     },
+    "popupSettingsReportIssue": {
+        "message": "Report Issue"
+    },
+    "popupSettingsSubmitReport": {
+        "message": "Submit Report"
+    },
+    "popupReportIssueSiteLabelHeadline": {
+        "message": "I’m having issues with this website"
+    },
+    "popupReportIssueSituationHeadline": {
+        "message": "Issue situation"
+    },
+    "popupReportIssueCaseOne": {
+        "message": "Email mask was not accepted by website"
+    },
+    "popupReportIssueCaseTwo": {
```

Comment:
Add a comment to say Do Not translate the product name. Check examples of the same instruction in this file so they stay consistent.

Reply:
Got it! Thanks for the heads up.